en/pro/install-license.md

```diff
@@ -0,0 +1,17 @@
+% Install License File
```

Comment:
This page is a duplicate of https://github.com/rundeck/docs/blob/3.0.x/en/administration/configuration/license.md

Reply:
Ok I will remove it

Reply:
done